### PR TITLE
change screen position

### PR DESCRIPTION
**Pull request by watasuke102**
The screen is moved top when the back screen was clicked.

### CODE REVIEW

**Comment by Aki-7** [comment]
Thanks for your first PR!!!
You are correctly replacing the view stacks, but you need to re-draw them after replacing the view stacks.

Try this on your zmonitor,

1. Run two clients,
```
$ weston-flower
```
```
$ weston-terminal
```

2. Hung one of them,
```
$ ps a | grep weston-terminal
$ kill -s SIGSTOP <pid>
```

3. Try to move the window of a stopped process. 

That should not be possible, but we would like to.
It's hard to notice this bug because most applications send some requests to commit a new drawing immediately after receiving a keyboard focus event. But some clients may not send any request after receiving keyboard focus event, and even so, we need to move the window forward.

So, in conclusion, the framebuffer needs to be updated.

**Comment by Aki-7** [comment]
And I recommend you to create a new branch on your fork.

**Review by Aki-7 — APPROVED** [review]
LGTM!!!
🎉 🎉 🎉 🎉 🎉 🎉 🎉
